web_ui/src/pages/annotator/annotation/list-item-grid.component.tsx

```diff
@@ -121,23 +78,8 @@ const Labels = ({ children, ...otherProps }: FlexProps) => {
     );
 };
 
-const ListMenu = ({ children, ...otherProps }: FlexProps) => {
-    const containerRef = useRef<null>(null);
-    const unwrapContainerRef = useUnwrapDOMRef(containerRef);
-    const { isHovered, setIsHovered } = useListItemGridContext();
-
-    if (!isHovered) {
-        return <></>;
-    }
-
-    return (
-        <>
-            <Flex {...otherProps} gridArea={'list-menu'} ref={containerRef}>
-                {children}
-            </Flex>
-            <InteractOutside containerRef={unwrapContainerRef} onClickOutside={() => setIsHovered(false)} />
-        </>
-    );
+const ListMenu = ({ children }: FlexProps) => {
```

Comment:
Could you clarify the changes here? Perhaps we can inline this `Flex` component as it no longer has it's own behaviour? 

Edit: on second look: it seems like we now use css's over styles?
I'm wondering though why we had it like this in the first place @camiloHimura can you double check this?